indico/modules/rb/client/js/common/bookings/BookingDetails.jsx

```diff
@@ -526,6 +543,22 @@ class BookingDetails extends React.Component {
               <Translate>Booking Details</Translate>
             </span>
             <span styleName="booking-status">{this.renderBookingStatus()}</span>
+            <span styleName="clipboard-copy">
+              <Popup
```

Comment:
I'd move this whole thing into a separate component (in indico/web/client/js/react/components/) so you can just do something like this here:

    <ClipboardButton text={externalDetailsURL} />